src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -81,9 +107,9 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
             return;
         }
 
-        if (empty($direction) && isset($this->properties[$property])) {
+        if (empty($direction) && null !== $defaultDirection = $this->properties[$property]['default_direction'] ?? null) {
```

Comment:
Can you explain the simple `=` declaration? I don't get how `null !== $defaultDirection = null` works.